extensions/2.0/Vendor/NV_materials_mdl/README.md

```diff
@@ -73,10 +73,11 @@ All modules that are referenced by the function calls in the `functionCalls` arr
 |:---------|:-----|:------------|:---------|
 | `uri` | `string` | The URI (or IRI) of the MDL module. | No |
 | `bufferView` | `integer` | The index of the buffer view that contains the MDL module. | No |
+| `mimeType` | `string` | The MDL module's media type. | No |
 | `modulePath` | `string` | Relative path of the module. | No |
 | `name` | `string` | glTF name of the module. | No |
 
-Each module defines either the `uri` property, which contains the URI (or IRI) of an external MDL module file or a data-URI with embedded data, or the `bufferView` property, which points to a buffer view object that contains an MDL module. The media type of a data-URI containing an MDL module must be `application/vnd.mdl`. If a data-URI or buffer view object is used, then the `modulePath` property must be defined, which specifies a path relative to the glTF file that is used for importing by other modules, e.g., if an MDL module that has the path `./my/embedded/module.mdl` relative to the glTF file is embedded into a buffer view object, then the `modulePath` property must have this path as its value. If `uri` contains a valid path to an external MDL module file, then `modulePath` must not be defined.
+Each module defines either the `uri` property, which contains the URI (or IRI) of an external MDL module file or a data-URI with embedded data, or the `bufferView` property, which points to a buffer view object that contains an MDL module. When the `bufferView` property is defined, then the `mimeType` property must also be defined. The module format must match the `mimeType` property when the latter is defined. The media type of a data-URI containing an MDL module must be `application/vnd.mdl`. If a data-URI or buffer view object is used, then the `modulePath` property must be defined, which specifies a path relative to the glTF file that is used for importing by other modules, e.g., if an MDL module that has the path `./my/embedded/module.mdl` relative to the glTF file is embedded into a buffer view object, then the `modulePath` property must have this path as its value. If `uri` contains a valid path to an external MDL module file, then `modulePath` must not be defined.
```

Comment:
> The media type of a data-URI containing an MDL module must be `application/vnd.mdl`

This language limits only data-URIs to the specific media type implying that regular URIs may have arbitrary types.

Please remove this sentence and add a new paragraph after this one with something like
> This extension currently supports only MDL modules of `application/vnd.mdl` media type.

Same comment for BSDF measurements.

Reply:
I added the paragraphs to the MDL moduels and MBSDFs sections.